cylc/flow/task_pool.py

```diff
@@ -1268,24 +1311,31 @@ def force_trigger_tasks(self, items, reflow=False):
         n_warnings, task_items = self.match_taskdefs(items)
         flow_label = self.flow_label_mgr.get_new_label()
         for name, point in task_items.keys():
-            # Already in pool? Keep merge flow labels.
-            itask = self.get_task(name, point, flow_label)
-            if itask is None:
-                # Spawn with new flow label.
-                itask = self.spawn_task(name, point, flow_label, reflow=reflow)
+            # Already in pool? Keep it and merge flow labels.
+            itask = self.get_task_main(name, point, flow_label)
             if itask is not None:
+                # Trigger existing task proxy
+                LOG.critical('setting %s ready to run', itask)
+                itask.is_manual_submit = True
+                itask.reset_try_timers()
                 # (If None, spawner reports cycle bounds errors).
-                itask.manual_trigger = True
                 if itask.state.reset(TASK_STATUS_WAITING):
                     self.data_store_mgr.delta_task_state(itask)
-                LOG.critical('setting %s ready to run', itask)
-                itask.state.set_prerequisites_all_satisfied()
-                self.data_store_mgr.delta_task_prerequisite(itask)
-                self.data_store_mgr.delta_task_outputs(itask)
```

Comment:
I guess the change and outputs/prereqs on force trigger doesn't happen in SOD?

I notice these are still called elsewhere..

Reply:
Hmmm, forgotten the details, I'd better check this change.

Reply:
(I remember now: force trigger means "run even though prerequisites are not satisfied" - so why pretend they were satisfied? ... hence, there is no data-store update to do here).